src/wp-includes/functions.php

```diff
@@ -3358,14 +3358,15 @@ function wp_get_image_mime( $file ) {
 		 * @todo check the third 4-byte token "[major_brand]", for "avif" or "avis" .
```

Comment:
TODO can be removed

Reply:
replaced the todo with a link to the specification from your previous comment.